src/java.net.http/share/classes/jdk/internal/net/http/hpack/QuickHuffman.java

```diff
@@ -732,7 +732,8 @@ public boolean write(ByteBuffer destination) {
                     if (c > 255) {
                         throw new IllegalArgumentException("char=" + ((int) c));
                     }
-                    long len = codeLengthOf(c);
+                    int len = codeLengthOf(c);
+                    assert len >= 0;
```

Comment:
This `assert` could be better if placed into `codeLengthOf`, which is called from one more place.